src/main/java/frc/robot/flywheel/FlywheelSubsystem.java

```diff
@@ -14,33 +18,43 @@ public class FlywheelSubsystem extends SubsystemBase {
      * Total: 2 TalonFX motors
      */
 
+    public TalonFX masterMotor;
```

Comment:
Good practice to make the hardware private so only the subsystem has access to them.

Reply:
kk